### PR TITLE
Local session

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: Session.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+'use strict';
+
+import SessionService from './SessionService';
+
+export default class Session {
+  constructor() {
+    this._service = new SessionService();
+  }
+
+  async refresh() {
```
Comment (mattcollier) [inline Session.js]: I don't know what the purpose of this is, but why is it async?

File: Session.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    return true;
+  }
+
+  async get() {
+    const data = await this._service.get();
```
Comment (mattcollier) [inline Session.js]: can we just 
```
return this._service.get();
```

File: Session.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    return data;
+  }
+
+  async end() {
+    const success = await this._service.logout();
```
Comment (mattcollier) [inline Session.js]: again, just return a promise (which is what happens anyway)?

File: Session.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import SessionService from './SessionService';
```
Comment (mattcollier) [inline Session.js]: What is the value of this Session class?  It doesn't look like it's saving state or anything.  Why not access the service directly?

**Review by mattcollier — CHANGES_REQUESTED** [review]
The date on the license file needs to be updated.

**Review by dlongley — APPROVED** [review]
LGTM other than it needs tests.

**Pull request merged**